docs/dev-guide/3.0-development/index.rst

```diff
@@ -0,0 +1,17 @@
+.. warning::
+    All documents within the 3.0-Development section should be considered temporary.
+
+
+3.0 Development
+=================
```

Comment:
2 less `=` to match length of section name  but *really* don't know if it's important.